### PR TITLE
php71-memcached

### PR DESCRIPTION
**Pull request by necromant2005**
php 7.1 memcached extension (just checked it works for the last versions available in homebrew)

### CODE REVIEW

File: Formula/php71-memcached.rb [hunk 1]
```diff
@@ -0,0 +1,167 @@
+require File.expand_path("../../Abstract/abstract-php-extension", __FILE__)
+
+class Php70Memcached < AbstractPhp70Extension
```
Comment (alcaeus) [inline Formula/php71-memcached.rb]: Needs to be `class Php71Memcached < AbstractPhp71Extension`

File: Formula/php71-memcached.rb [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+  init
+  desc "Memcached via libmemcached library"
+  homepage "https://pecl.php.net/package/memcached"
+  head "https://github.com/php-memcached-dev/php-memcached.git", :branch => "php7"
+
+  option "with-sasl", "Build with sasl support"
+
+  depends_on "pkg-config" => :build
+  depends_on "php70-igbinary"
```
Comment (alcaeus) [inline Formula/php71-memcached.rb]: `depends_on "php71-igbinary"`

**Comment by alcaeus** [comment]
@necromant2005 can you make the changes suggested above or should I open a new pull request?

**Comment by lucasmichot** [comment]
Your PR was fixed with https://github.com/Homebrew/homebrew-php/pull/3798/commits/bc59da978b053bf8046c9f2687690f841952d03d @necromant2005

File: Formula/php71-memcached.rb (hunk 1 above)
Comment (javian) [inline Formula/php71-memcached.rb]: This has now been corrected in later commits.

**Comment by sergeyz** [comment]
https://github.com/Homebrew/homebrew-php/blob/master/Formula/php71-igbinary.rb already in master and all remarks were fixed. Can we merge this, @javian ?

**Comment by javian** [comment]
This PR will be superceded by the 3.0.0 release - closing.

**Pull request closed without merging**